src/main/java/uk/nhs/hee/tis/trainee/sync/facade/PlacementEnricherFacade.java

```diff
@@ -417,4 +558,66 @@ private String getSiteName(Site site) {
   private String getSiteLocation(Site site) {
     return site.getData().get(SITE_LOCATION);
   }
+
+  /**
+   * Get the Placement Specialty ID from the placement.
+   *
+   * @param placement The placement to get the placmeent specialty id from.
+   * @return The placement specialty id.
+   *
+   *          Note: since only one primary specialty exists per placement,
+   *          we use placementId as placement specialty ID
+   */
+  private String getPlacementSpecialtyId(Placement placement) {
+    return placement.getTisId();
```

Comment:
Just referencing for the other comment `placement.id != placementspecialty.id`

Reply:
I renamed the method, rewrote the javadoc comment to be more explainatory and renamed the variable where the method is used (and added a comment for extra clarity) to avoid confusion. 
Its usage on line 206 appear fine to me.